typescript/cli/ci-test.sh

```diff
@@ -98,7 +100,7 @@ yarn workspace @hyperlane-xyz/cli run hyperlane deploy core \
     $(if [ "$HOOK_FLAG" == "true" ]; then echo "--hook ${EXAMPLES_PATH}/hooks.yaml"; fi) \
     --ism ${EXAMPLES_PATH}/ism.yaml \
     --out /tmp \
-    --key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 \
+    --key $ANVIL_KEY \
```

Comment:
sweet thanks - could you also update the other instances below where we send the test messages